tests/test_elasticsearch.py

```diff
@@ -97,114 +95,3 @@ def test_new_search_doesnt_show_unpublished(db, products_with_mixed_publishing,
     published_products = STOREFRONT_PRODUCTS - PRODUCTS_TO_UNPUBLISH
     found_products = execute_search(client, PHRASE_WITH_RESULTS)
     assert published_products == set(found_products)
-
-
-def execute_dashboard_search(client, phrase):
-    response = client.get(reverse("dashboard:search"), {"q": phrase})
-    assert phrase == response.context["query"]
-    found_prod = {p.pk for p in response.context["products"]}
-    found_users = {p.pk for p in response.context["users"]}
-    found_orders = {p.pk for p in response.context["orders"]}
-    return found_prod, found_users, found_orders
-
-
-@pytest.mark.integration
-@pytest.mark.vcr(record_mode="once", match_on=MATCH_SEARCH_REQUEST)
```

Comment:
Don't forget to remove the cassettes in that case 👍

Reply:
Done 👍